com.qzandronum.Q-Zandronum.yaml

```diff
@@ -0,0 +1,185 @@
+app-id: com.qzandronum.Q-Zandronum
+runtime: org.kde.Platform
```

Comment:
are you sure the freedesktop sdk isn't enough.

Reply:
Yes, as Doomseeker that Q-Zandronum uses use QT.